plugins/orchestrator-backend/src/types/pagination.ts

```diff
@@ -0,0 +1,17 @@
+import { Request } from 'express-serve-static-core';
+
+export interface Pagination {
+  offset?: number;
+  limit?: number;
+  order?: string;
+  sortField?: string;
+}
+
+export function buildPagination(req: Request): Pagination {
+  return {
+    offset: req.query.pageNumber ? Number(req.query.pageNumber) : 0,
+    limit: req.query.pageSize ? Number(req.query.pageSize) : 10,
```

Comment:
We'll get a `NaN` value if `pageNumber` or `pageSize` is not a number string.
I think it would be enough if we use 
```ts
offset: isNaN(req.query.pageNumber) ? 0 : Number(req.query.pageNumber);
```
Please evaluate it.

Reply:
[17b78c8](https://github.com/janus-idp/backstage-plugins/pull/1164/commits/17b78c829f840d61a494fbcf0aa2d5524176e852)